### PR TITLE
Docker

### PR DESCRIPTION
**Pull request by Hgmoa**

### CODE REVIEW

File: src/main/java/daw/spring/Application.java [hunk 1]
```diff
@@ -33,7 +33,14 @@
     public static final Path UPLOADED_FILES_PATH = Paths.get("uploaded");
     public static final Path USERS_IMAGES_PATH = UPLOADED_FILES_PATH.resolve("users");
     public static final Path PRODUCTS_IMAGES_PATH = UPLOADED_FILES_PATH.resolve("products");
-    private final Logger log = LoggerFactory.getLogger(getClass());
+    private static final Logger log = LoggerFactory.getLogger(Application.class);
+
+    static {
+        log.info(UPLOADED_FILES_PATH.toAbsolutePath().toString());
```
Comment (Hgmoa) [inline src/main/java/daw/spring/Application.java]: MAJOR Invoke method(s) only conditionally. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2629)

File: src/main/java/daw/spring/Application.java [hunk 2]
```diff
@@ -33,7 +33,14 @@
...
+        log.info(USERS_IMAGES_PATH.toAbsolutePath().toString());
```
Comment (Hgmoa) [inline src/main/java/daw/spring/Application.java]: MAJOR Invoke method(s) only conditionally. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2629)

File: src/main/java/daw/spring/Application.java [hunk 3]
```diff
@@ -33,7 +33,14 @@
...
+        log.info(PRODUCTS_IMAGES_PATH.toAbsolutePath().toString());
```
Comment (Hgmoa) [inline src/main/java/daw/spring/Application.java]: MAJOR Invoke method(s) only conditionally. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2629)

**Comment by Hgmoa** [comment]
SonarQube analysis reported 3 issues
* ![MAJOR][MAJOR] 3 major

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by CVelazquezURJC** [comment]
very good job

**Comment by JorgeGalingP** [comment]
:v: :new_moon_with_face: :ok_hand:

**Pull request merged**